### PR TITLE
feed: Make sure to filter out cards without custom titles

### PR DESCRIPTION
**Pull request by smspillaz**
We were not following the spec here - instead we were falling back
for all apps that supported the interface, which is all apps.

https://phabricator.endlessm.com/T19108

### CODE REVIEW

File: search-provider/eks-discovery-feed-provider.c [hunk 1]
```diff
@@ -790,41 +775,34 @@ content_article_card_descriptions_cb (GObject *source,
...
+
+      /* Start building up object */
+      g_variant_builder_open (&builder, G_VARIANT_TYPE ("a{ss}"));
 
+      add_key_value_pair_to_variant (&builder, "title", title);
+      add_key_value_pair_to_variant (&builder, "synopsis", "");
       add_key_value_pair_from_model_to_variant (model, &builder, "last-modified-date");
       add_key_value_pair_from_model_to_variant (model, &builder, "thumbnail-uri");
       add_key_value_pair_from_model_to_variant (model, &builder, "ekn-id");
+      add_author_from_model_to_variant (model, &builder, "author");
```
Comment (cosimoc) [inline search-provider/eks-discovery-feed-provider.c]: This change does not look related.

**Comment by cosimoc** [comment]
After talking to @erikos this morning, I'm holding on this change for now. I will follow up on the ticket.

File: search-provider/eks-discovery-feed-provider.c (hunk 1 above)
Comment (smspillaz) [reply search-provider/eks-discovery-feed-provider.c]: Oops, the perils of copy-and-paste

**Comment by smspillaz** [comment]
Closing this for now so that it doesn't cause confusion
